bin/generate-latest-gce-images

```diff
@@ -4,7 +4,7 @@
 require 'json'
 
 DEFAULT_GCE_PROJECT_ID = 'eco-emissary-99515'
-DEFAULT_IMAGE_PREFIXES = 'bastion nat tfw'
+DEFAULT_IMAGE_PREFIXES = 'bastion'
```

Comment:
I assume this is a change unrelated to to warmer and just changes made to NAT or `tfw`

Reply:
I was getting an image lookup failure and realized that this list was longer than necessary :man_shrugging: